.rubocop.yml

```
@@ -0,0 +1,12 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-rspec
+
+AllCops:
+  NewCops: enable
+  TargetRubyVersion: 2.4
```

2.5 is the lowest supported version
```suggestion
  TargetRubyVersion: 2.5 # lowest supported version
```